server/repository/src/db_diesel/assets/asset_log_row.rs

```diff
@@ -51,6 +51,7 @@ pub enum AssetLogStatus {
     FunctioningButNeedsAttention,
     NotFunctioning,
     Decommissioned,
+    Unserviceable,
```

Comment:
I think this will need to be added to the postgres enum for Asset Log Status using a migration.

See: https://github.com/msupply-foundation/open-msupply/blob/702223413b605fd2af74b6692f591e6c96cb330c/server/repository/src/migrations/v2_00_00/assets/asset_log.rs#L9

Reply:
Yeah good point, will update...

Reply:
Done, confirmed now working with Postgres: cbd3eb194f